_data/mentors.yml

```diff
@@ -61,11 +61,11 @@
     Mulle meeldib käsitööga tegeleda: kudumine, heegeldamine, kunst, õmblus, tikkand jms. 
 
 - name: Aimar Tuul
-  age: 26
+  age: 27
   photo: aimar_tuul_mentor.jpg
   speciality: IT-süsteemide arendus
   teams:  ['Tarkvara']
-  term: 4
+  term: 5 
```

Comment:
dat trailing whitespace. Do you even format?

Reply:
unlucky, ei märganud